hieroplatform/src/test/java/org/hiero/sketch/IntConverter.java

```diff
@@ -0,0 +1,13 @@
+package org.hiero.sketch;
```

Comment:
This class should not be needed; it may signal a bug elsewhere.
What happens without it?